jira.go

```diff
@@ -231,6 +234,10 @@ func (c *Client) Do(req *http.Request, v interface{}) (*Response, error) {
 		// Open a NewDecoder and defer closing the reader only if there is a provided interface to decode to
 		defer httpResp.Body.Close()
 		err = json.NewDecoder(httpResp.Body).Decode(v)
+
+		if err != nil {
+			log.Printf("error JSON encoding body: %v", httpResp.Body)
```

Comment:
This leads to a side effect and is not necessary, because a couple of lines after in https://github.com/andygrunwald/go-jira/blob/master/jira.go#L246 the `err` is returned.
I would vote for a removal. Or is there any special reason for it?

Reply:
Very often I can't understand what kind of error jira thrown when I ask for some data. Logging the answer helped to investigate it. I think that this debugging approach will be helpful for others in interacting with jira.